<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>commonAddDlg</class>
 <widget class="QDialog" name="commonAddDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLabel,
QPushButton{
font-size:12px;
font-family:&quot;微软雅黑&quot;;
}
#titleWidget{
background:#ffffff;
border-top-left-radius:4px;
border-top-right-radius:4px;
}
#widget{
background:#eeeeee;
border-bottom-left-radius:4px;
border-bottom-right-radius:4px;
}
#titleLabel{
color:rgba(50,52,52,1);
}
#tab1{
border:1px solid #cccccc;
border-top-left-radius:14px;
border-bottom-left-radius:14px;
border-right:none;
background:transparent;
color:#888888;
}
#tab1:checked{
background:#cccccc;
color:#222222;
}
#tab2{
border:1px solid #cccccc;
border-top-right-radius:14px;
border-bottom-right-radius:14px;
background:transparent;
color:#888888;
}
#tab2:checked{
background:#cccccc;
color:#222222;
}
#confirmBtn{
background:#ffd500;
border-radius:2px;
color:#333333;
}
#confirmBtn:pressed{
margin:1px;
}
#cancelBtn{
background:#ffffff;
border-radius:2px;
color:#333333;
}
#cancelBtn:pressed{
margin:1px;
}
#listView{
border:1px solid #cccccc;
}
#selectTitleWidget{
border:1px solid #cccccc;
background:#ffffff;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="titleWidget" native="true">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>32</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>32</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="titleLabel">
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>20</number>
      </property>
      <property name="leftMargin">
       <number>100</number>
      </property>
      <property name="topMargin">
       <number>20</number>
      </property>
      <property name="rightMargin">
       <number>100</number>
      </property>
      <property name="bottomMargin">
       <number>40</number>
      </property>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="tab1">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="tab2">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>28</height>
           </size>
          </property>
          <property name="text">
           <string/>
          </property>
          <property name="checkable">
           <bool>true</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QStackedWidget" name="stackedWidget_2">
        <property name="minimumSize">
         <size>
          <width>320</width>
          <height>183</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>320</width>
          <height>183</height>
         </size>
        </property>
        <widget class="QWidget" name="deviceListPage" native="true"/>
        <widget class="QWidget" name="sceneListPage"/>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <spacer name="horizontalSpacer_3">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="confirmBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>确认</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cancelBtn">
          <property name="minimumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>80</width>
            <height>26</height>
           </size>
          </property>
          <property name="text">
           <string>取消</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
